authorization/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+
+    // TODO: fix this hack and invoke the protoc during the build via docker; maybe convert to a replicate task
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
```

Comment:
Why are you needing this for now?

Reply:
Possible this is getting picked up from some prelim work I did on adding support for a variety of authentication handlers, which lived in a shared branch we used.

Reply:
I did this to generate a single jar that I could easily invoke from a shell script. To invoke protoc you need flags like this:
```
PATH=.:$PATH protoc --service-auth-wiring_out=authorization/src/main/java/ -I proto/proto-backplane-grpc/src/main/proto/  proto/proto-backplane-grpc/src/main/proto/deephaven/proto/console.proto
```

Which will automatically look for an executable called `protoc-gen-service-auth-wiring` on the path. To ease my pain, these were shell scripts that used the jar-jar. Protoc expects to communicate in stdin vs stdout and I couldn't figure out how to do that via gradle -- so this was my cheap work around given that Colin is going to direct me how to automate the generation.

Reply:
Probably could use the application plugin to achieve this too, it will give you a zip/tarball you can use as input into the docker container, and a script inside there which can be executed.